src/fusion_power_plant.h

```diff
@@ -47,9 +47,6 @@ class FusionPowerPlant : public cyclus::Facility  {
 
   #pragma cyclus note {"doc": "A stub facility is provided as a skeleton " \
                               "for the design of new facility agents."}
-  //Functions:
-  /// A verbose printer for the FusionPowerPlant
-  virtual std::string str();
```

Comment:
Does this mean you're just relying on the default `str()` function for `Facility`?  Or do you perhaps want to comment this out and revisit later?

Reply:
I think it's pretty common to implement this to describe the specific details of this kind of facility.

Reply:
I removed this because of a conversation i thought we had on the original PR about this function not being really used for anything. I can add it back, though, especially since I can't find that conversation anymore...

Reply:
It's not strictly necessary, but it is useful to have a way to print out information about a prototype/facility and its configuration parameters.